python/lsst/ip/diffim/subtractImages.py

```diff
@@ -193,12 +193,33 @@ class AlardLuptonSubtractBaseConfig(lsst.pex.config.Config):
         doc="Minimum signal to noise ratio of detected sources "
         "to use for calculating the PSF matching kernel."
     )
+    detectionThresholdMax = lsst.pex.config.Field(
+        dtype=float,
+        default=500,
+        doc="Maximum signal to noise ratio of detected sources "
+        "to use for calculating the PSF matching kernel."
```

Comment:
This is a good parameter to have available. The default value is OK, and it would be best for each instrument to have this value configured accordingly.